File: DfmCore/DictionaryInfo.cs
using DFMServer;

namespace DfmCore
{
    public class DictionaryInfo
    {
        public DictionaryInfo(Dictionary7 dictObj)
        {
            DSN       = dictObj.DSN;
            Version   = dictObj.Version;
            UserName  = dictObj.CurrentUser;
            UserGroup = dictObj.CurrentGroup;

            dictObj.GetUserAttributes(
                UserName,
                out var userFullName,
                out var description,
                out var isLegalResponsible);

            UserFullName = userFullName;
            Description = description;
            IsLegalResponsible = isLegalResponsible;
        }

        public string DSN                { get; }
        public string Version            { get; }
        public string Description        { get; }
        public string UserName           { get; }
        public string UserFullName       { get; }
        public string UserGroup          { get; }
        public bool   IsLegalResponsible { get; }
    }
}

File: DfmWeb.App/App/models/dictionaryInfo.js
export default class DictionaryInfo {
    constructor(obj) {
        this.dsn = obj ? obj.dsn : '';
        this.version = obj ? obj.version : '';
        this.description = obj ? obj.description : '';
        this.userName = obj ? obj.userName : '';
        this.userFullName = obj ? obj.userFullName : '';
        this.userGroup = obj ? obj.userGroup : '';
        this.isLegalResponsible = obj ? obj.isLegalResponsible : '';
    }
}